### DISCUSSION TITLE
Stable release page security issue [Wiki]

### DISCUSSION THREAD

**Discussion by sreimers** [baresip/baresip#2287] [General]
The librem download link on https://github.com/baresip/baresip/wiki/Install:-Stable-Release page was compromised on `Mon Oct 3 22:04:28 2022 +0300.`
![Bildschirmfoto vom 2022-10-26 17-13-45](https://user-images.githubusercontent.com/296762/198067114-14b3059e-de0d-4aff-bbce-1d7eba1c33ba.png)
![Bildschirmfoto vom 2022-10-26 17-13-58](https://user-images.githubusercontent.com/296762/198067155-32b3c881-4905-4af0-82af-f4e6b0ddc769.png)

Looks like the user is already deleted by github. I just reverted the change and have ensured that no further changes have been made.
Should we restrict the wiki? Or at least move the install instructions to the repo?

**Comment by robert-scheck** [comment]
While I appreciate and support the actions taken, may I kindly ask for some wiki edit permission? I e.g. update [supported platforms
](https://github.com/baresip/baresip/wiki/Supported-platforms) every now and then.

**Reply by cspiel1** [reply]
@sreimers Is it possible to do wiki updates via pull requests?

**Reply by sreimers** [reply]
It's a hidden repo, so I think there is no public way. But maybe we can add a wiki restricted documentation team role.

**Reply by robert-scheck** [reply]
Any status update?

**Reply by sreimers** [reply]
I just send you a Invite

**Reply by robert-scheck** [reply]
Accepted it, thank you!
